src/__builtins.di

```diff
@@ -68,4 +68,14 @@ version (DigitalMars)
     alias __builtin_fabs  = core.stdc.math.fabs;
     alias __builtin_fabsf = core.stdc.math.fabsf;
     alias __builtin_fabsl = core.stdc.math.fabsl;
+
+    import core.bitop;
+
+    alias __builtin_bswap32 = core.bitop.bswap;
+    alias __builtin_bswap64 = core.bitop.bswap;
+
+    // Stub these out to no-ops
+    enum  __builtin_constant_p = 0; // should be something like __traits(compiles, enum X = expr)
```

Comment:
signature is wrong should be `int   __builtin_constant_p( int value) { return 0; }` (with a pessimism that no value is know at compile time).

Reply:
actually it should be a template, as the argument can be of any type

Reply:
Do we support `enum foo = (int) => 0;`?